### ISSUE TITLE
[BUG] Reversed flashcards inconsistent order

### ISSUE THREAD

**Issue by AndriiPetrovDev** [st3v3nmw/obsidian-spaced-repetition#875]
I have a bug with quite simple data. I see that the Plugin mixes order of questions and answers while it should be constantly reversed. In straight order everything works fine

```
question1 ::: answer1 
question2 ::: answer2
question3 ::: answer3
question4 ::: answer4
#flashcards
```

<img width="1048" alt="SCR-20240212-icog" src="https://github.com/st3v3nmw/obsidian-spaced-repetition/assets/13622708/831ad604-52ec-4235-a5f0-8e66e067fd08">
<img width="1345" alt="SCR-20240212-iclx" src="https://github.com/st3v3nmw/obsidian-spaced-repetition/assets/13622708/175dbcd8-7e12-4101-9342-36655f84a26b">

-   OS: MacOS 14.2.1
-   Obsidian version: : V1.5.3
-   Plugin version: 1.11.1

**Comment by ronzulu** [comment]
Hi there, could you please list the order that you are expecting, e.g.

1. question1 -> answer1
2. answer1 -> question1

Also please send a screenshot of the settings for the flashcard review order

![image](https://github.com/st3v3nmw/obsidian-spaced-repetition/assets/75528127/89501f2d-3289-4166-8b51-9457156836cb)

**Comment by AndriiPetrovDev** [comment]
Hi @ronzulu 
I expect this result for all cards:
**answerN -> questionN**\
here is my settings just to be sure that you have all needed information
![SCR-20240213-pxvg](https://github.com/st3v3nmw/obsidian-spaced-repetition/assets/13622708/ccd66eff-355f-4c43-9a11-4701dc216792)

**Comment by AndriiPetrovDev** [comment]
Unfortunately multyline cards works the same
if I set 
<img width="866" alt="image" src="https://github.com/st3v3nmw/obsidian-spaced-repetition/assets/13622708/163e5328-affa-45f8-b9d4-e23a37b09135">
the order is:

question1 -> answer1
answer1 -> question1
question2 -> answer2
answer2 -> question2
question3 -> answer3
answer3 -> question3
question4 -> answer4
answer4 -> question4

**Comment by ronzulu** [comment]
Hi @AndriiPetrovDev 

Thanks for the info. So can I check, you just want the following:

```
answer1 -> question1
answer2 -> question2
answer3 -> question3
answer4 -> question4
```

I think the problem is that "inline reversed" is not an accurate name on the settings page:
![image](https://github.com/st3v3nmw/obsidian-spaced-repetition/assets/75528127/536a6cfa-00fc-4bd6-a359-54b3f49bb0a0)

In the user manual it is also called "reversed", but the description is accurate:
https://www.stephenmwangi.com/obsidian-spaced-repetition/flashcards/#single-line-reversed

![image](https://github.com/st3v3nmw/obsidian-spaced-repetition/assets/75528127/5271486f-5c62-456f-a27b-b08f61b410f7)

The inaccurate naming of "reversed" also caused confusion for #865

Cheers
Ronny

**Comment by ronzulu** [comment]
I've created a new issue regarding the terminology at #876 

I can see that it might be beneficial to include new card types called "reversed" that are just reversed rather than bidirectional.

Out of curiosity, why do you prefer something like:
`Question <== Answer`

Over existing functionality of:
`Answer :: Question`

Cheers
Ronny

**Comment by AndriiPetrovDev** [comment]
@ronzulu   Yeah, it will be very beneficial for language learners, Because both directions "native"->"translation" and  "translation" <-"native" is helpful in learning. Sometimes you just translate foreign word but in other case you can learn how to express your thoughts in different language
